### PR TITLE
Update output filename in packaging-project

### PR DESCRIPTION
**Pull request by daxamin**
#999

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,4 @@ build/
 __pycache__
 .DS_Store
 /locales/
+.vscode
```
Comment (webknjaz) [inline .gitignore:8]: This doesn't seem related to changing the docs.

File: source/tutorials/packaging-projects.rst [hunk 2]
```diff
@@ -524,14 +524,14 @@ After the command completes, you should see output similar to this:
...
+    Uploading example-package-YOUR-USERNAME-HERE-0.0.1-py3-none-any.whl
     100%|█████████████████████| 4.65k/4.65k [00:01<00:00, 2.88kB/s]
-    Uploading example_package_YOUR_USERNAME_HERE-0.0.1.tar.gz
+    Uploading example-package-YOUR-USERNAME-HERE-0.0.1.tar.gz
     100%|█████████████████████| 4.25k/4.25k [00:01<00:00, 3.05kB/s]
 
 
 Once uploaded your package should be viewable on TestPyPI, for example,
-https://test.pypi.org/project/example-pkg-YOUR-USERNAME-HERE
+https://test.pypi.org/project/example-package-YOUR-USERNAME-HERE
```
Comment (meowmeowmeowcat) [inline source/tutorials/packaging-projects.rst:534]: Could you also modify this link in `conf.py` so  linkcheck can ignore it correctly?

**Review by meowmeowmeowcat — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (daxamin) [reply .gitignore]: removed

File: source/tutorials/packaging-projects.rst (hunk 2 above)
Comment (daxamin) [reply source/tutorials/packaging-projects.rst:534]: done

**Review by webknjaz — APPROVED** [review]

**Comment by bhrutledge** [comment]
@webknjaz Looks like `linkcheck` is failing for reasons unrelated to this PR (http://www.pyinstaller.org/ isn't responding). Is there a way to merge this PR anyway?

**Comment by webknjaz** [comment]
I don't have override privileges. Somebody needs to fix that link first.

**Pull request merged**